go.mod

```diff
@@ -10,3 +10,5 @@ require (
 )
 
 require filippo.io/edwards25519 v1.1.0 // indirect
+
+replace github.com/muir/sqltoken => github.com/vinovest/sqltoken v0.0.6-sqlx
```

Comment:
Any reason to use replace instead of updating the code to use your fork ?

@mrj0

Reply:
ah well that was the state of the art 6 months ago

Reply:
I made changes that he merged into the tokenizer and later versions make use of the latest. This change was followed up by a lot of other improvements as well.

Reply:
So now, you could use again muir/sqltoken directly?

Reply:
It does already. You’re commenting on an old pr but work continued after this